[03:26:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[03:26:53] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
